content/docs/0.16.x/manage/triggers/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Triggers
+description: Using triggers to kick off a sequence
+weight: 50
+keywords: [0.15.x-manage]
+aliases:
+---
+
+You can define *triggers* to kick off a sequence in response to specific events.
```

Comment:
Other OSS tools use the word "trigger" to denote starting from an external source.

You can begin Keptn sequences via the bridge UI, API or CLI. This page specifically talks about an internal sequence-to-sequence triggering mechanism _inside_ keptn.

Reply:
See previous comments.  This is fixed, sort of, for now.